src/users/dto/update-preferredMap.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { IsNotEmpty, IsNumber } from 'class-validator';
+
+export class UpdatePreferredMapDto {
+  @ApiProperty({ description: '0 - 카카오 / 1 - 구글 / 2 - 네이버' })
+  @IsNumber()
+  @IsNotEmpty()
+  preferredMap: number;
```

Comment:
```suggestion
  preferredMap: 0 | 1 | 2;
```

이렇게 타입 지정을 해두면 아래 400 예외처리를 if 문으로 구현하지 않아도 dto 단에서 0, 1, 2 가 아닌 값에는 400 에러를 반환할 겁니다.

Reply:
오옷 ! 바로 반영해보겠습니다